trim_lcnaf.rb

```diff
@@ -0,0 +1,24 @@
+require 'csv'
+require 'json'
+
```

Comment:
This is now using the Ruby CSV library to write out. The nice thing about this is that rows are represented as arrays so it's easy to read in values with something like: `value_a, value_b, value_c = row` which will assign value_a to row[0], value_b to row[1], and value_c to row[2].